python/lsst/jointcal/jointcal.py

```diff
@@ -430,7 +441,9 @@ def _do_load_refcat_and_fit(self, associations, defaultFilter, center, radius,
         # TODO: this should not print "trying to invert a singular transformation:"
         # if it does that, something's not right about the WCS...
         associations.associateCatalogs(match_cut)
-        self.metrics['associated_%s_fittedStars' % name] = associations.fittedStarListSize()
+        meas = Measurement(self.job.metrics['jointcal.associated_%s_fittedStars' % name],
```

Comment:
Is there not a one-line way to add a Measurement to a Job?

Reply:
I guess part of my point is that this makes taking `Measurements` more complicated (in developer brain space/time) than it maybe should be. I feel like we need something like this:

```
self.job.add_measurement('somename', value)
```

Reply:
OK.  I'll suggest a solution on the branch.

Reply:
I've implemented this in the task file.  I don't think it needs to be pushed upstream given they way it's used in `validate_drp`, but it could.

Reply:
Oh, I was thinking to have it upstream. How are we planning to make Measurements in other tasks?

Reply:
How about:

```
self.job.measurements.insert(Measurement('somename', quantity_or_dimensionless))
```